source/hiring/index.md

```diff
@@ -43,15 +39,15 @@ We want to build a team of Software Developers with a range of backgrounds and c
  - TeX/LaTeX rationalization &ndash; arXiv accepts the bulk of its content in TeX format. We have TeX or LaTeX source for 90% of our articles. However, compiling all the TeX source we into viewable papers (in PDF and increasingly also in HTML) can be difficult. We maintain a principle that we can rebuild any TeX submission accepted in the last 32 years. Preserving this capability, while being able to accept submissions that depend on the latest and greatest versions of TeX or LaTeX has become increasingly challenging. It is not unusual for a new release of texlive to include changes that cause older papers to no longer build properly. We have an opening for a <b>TeX/LaTeX</b> expert to rationalize our TeX pipeline. This task may include getting deeply involved in the TeX community and contributing to projects that support arXiv's goals.
 
-But if you're simply a software generalist, front-end, back-end, or whatever, that's fine too. As long as you have the requisite background (combination of education and experience <i>and can quickly learn new technologies</i> we encourage you to apply.
+But if you're simply a software generalist, front-end, back-end, or whatever, that's fine too. As long as you have the requisite background combination of education and experience <i>and can quickly learn new technologies</i> we encourage you to apply.
 
-We have three Software Developer listings corresponding to various levels of seniority. We may hire two developers each at the IV and V levels. Below are links to the job postings at Cornell:
+We have three Software Developer listings corresponding to various levels of seniority. We may hire more than one developer each for some of these positions. Below are links to the job postings at Cornell:
 
-1. Software Developer III
-1. Software Developer IV
-1. Software Developer V
+ - [Software Developer III](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
+ - [Software Developer IV](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
+ - [Software Developer V](https://cornell.wd1.myworkdayjobs.com/en-US/CornellCareerPage/?q=arxiv)
 
-### Program Manager
+### Project Manager
 
 We need a Program Manager/Project Manager to organize the arXiv CE effort. This job is modelled on the Program Manager roles found at a lot of modern tech companies like Microsoft and Google. The simple description is the PM does whatever needs to be done &ndash; short of writing code (although that's not out of the question), to support the goals of the project. A secondary description is to take some of the load for managing day to day details away from the Technial Director, so the TD can focus on higher level tasks. Some things the PM might do:
```

Comment:
If you change 'Program' to 'Project' in heading you also need to change two occurrences in this paragraph.